.github/renovate.json5

```diff
@@ -10,63 +10,80 @@
       "enabled": false,
       "matchManagers": ["regex"],
       "matchUpdateTypes": ["major"]
-    },{
+    },
+    {
       // Disable minor version update for previous Camunda releases which will only get patch updates.
       "enabled": false,
       "matchManagers": ["regex"],
       "matchPaths": [".env"],
-      "matchBaseBranches": ["/^stable\\/8.0/", "/^stable\\/8.1/", "/^stable\\/8.2/"],
+      "matchBaseBranches": ["/^stable\\/8\\..*/"],
       "matchUpdateTypes": ["minor"]
-    },{
-      "matchBaseBranches": ["/^stable\\/8.0/"],
-      "matchDepNames": ["camunda/operate"],
-      "allowedVersions": "<8.1.0",
     },
+    // Operate stable branch conditions
```

Comment:
TIL Json5 adds comment support ❤️  I went to order the entries by component to have a better overview if all is covered, let me know what you think.

Reply:
With all the at least major and minor version alignment across components, we may just have one entry per stable branch and actually add multiple `matchDepNames` entries? 🤔

Reply:
That comment support is quite nice.  we make heavier use of it in the helm chart :grin: .

> With all the at least major and minor version alignment across components, we may just have one entry per stable branch and actually add multiple matchDepNames entries? 🤔

I think it could work like that. When I code renovate configuration files, I often under-engineer it because testing renovate is pretty time-expensive. one bad regex can take hours to get right. I've made some progress in testing with regex101 and  using a forked repo  to prove that the PRs recommended don't unintentionally recommend bad upgrades, but it's still a pain :sweat: 

I'd say for now, lets just keep matchDepNames separate.